nsq/writer.py

```diff
@@ -131,7 +135,7 @@ def _pub(self, command, topic, msg, callback):
         conn.callback_queue.append(callback)
         cmd = getattr(protocol, command)
         try:
-            conn.send(cmd(topic, msg))
+            conn.send(cmd(topic, msg) if delay_ms is None else cmd(topic, delay_ms, msg))
```

Comment:
I tried my best not to modify much from original code. I can try cleaning it up.